### DISCUSSION TITLE
How to help with the website translation?

### DISCUSSION THREAD

**Discussion by KierPrev** [linuxmint/discussions#23] [Questions]
Not every user speaks English.  It is nice that the operating system is translated, but the website should be as well.  **At least the main page and the downloads page.**

The site should detect your country, and assign a default language based on that. Or maybe detect the language of your web-browser.

I can help with the spanish translation.

**Comment by carlosmintfan** [comment]
I can help with i18n (putting strings into translation function(s)) and maybe a translation system too.
